loans/models.py

```diff
@@ -1,13 +0,0 @@
-from django.db import models
-
-from books.models import Book
-from patrons.models import Patron
-
-
-# Create your models here.
-class Loan(models.Model):
```

Comment:
need to add class Meta
add relevant helping + error messages
add __str__ function for the admin